hash/to_flop.xml

```diff
@@ -14317,7 +14317,7 @@ Thanks to DCMOTO (http://dcmoto.free.fr) for info!
 	</software>
 
 	<software name="trivialpb" cloneof="trivialp">
-		<description>Trivial Pursuit (Compilation Top Adventure)</description>
+		<description>Trivial Pursuit (compilation Top Adventure)</description>
```

Comment:
Same with this one.

Reply:
I didn't see the "Compilation" word on the box (for the Top Arcade) neither inside (for the Top Adventure), that's why I removed it.
![image](https://user-images.githubusercontent.com/26376777/227733110-08dde928-0b7e-4a1a-96d8-700476b01624.png)
![image](https://user-images.githubusercontent.com/26376777/227733324-fd0935a5-664b-40c2-bcf9-dca9a2cca7f4.png)

Reply:
Top Adventure and Top Arcade should be turned into multi-part items containing the constituent disks anyway.

Reply:
I totally agree, I'll do it next time.